### DISCUSSION TITLE
[Kakao Tech Campus Step2] 2주차 회고

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#590] [5. 회고]
- [블로그 바로 가기](https://animoto1.tistory.com/entry/Kakao-Tech-Campus-Step2-2%EC%A3%BC%EC%B0%A8-%ED%9A%8C%EA%B3%A0) 
- 이번주는 헤카톤떄문에 개인공부를 많이 못한 거 같네요 ㅠㅠㅠ... 
- 담주도 화이팅!!!

**Comment by shkisme** [comment]
와 언제 봐도 정말 정리를 잘 하시는 것 같아요 !! 블로그 방문자수가 ㄷㄷ!
카테캠이랑 대회 전부 화이팅입니다~!!

**Reply by pykido** [reply]
감사합니다! 중간에 로그인 세션이 만료되어 내용이 다 날라간 탓에 지금 중간 부분을 다시 채우고 있네요 ㅠㅠ..
